assets/js/setup.tsx

```diff
@@ -0,0 +1,60 @@
+import Anchor from "anchor-js";
+import PrivateEye from "@18f/private-eye";
+import "simple-jekyll-search";
+
+// eslint-disable-next-line import/no-unresolved
+import { SimpleJekyllSearchGlobals } from "simple-jekyll-search-types";
```

Comment:
Does it make a difference if we use `import type` here?

```suggestion
import type { SimpleJekyllSearchGlobals } from "simple-jekyll-search-types";
```

Reply:
oh wow, TIL about `import type` --- thanks! PR to fix coming up shortly

Reply:
https://github.com/18F/identity-handbook/pull/237